azure-pipelines.yml

```diff
@@ -0,0 +1,164 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
```

Comment:
is this pipeline being used?

Reply:
It's demonstration pipeline that does with it.